Minimal patch to get chromium to compile with widevine support

https://bugs.gentoo.org/show_bug.cgi?id=547630

--- a/third_party/widevine/cdm/stub/widevine_cdm_version.h
+++ b/third_party/widevine/cdm/stub/widevine_cdm_version.h
@@ -10,6 +10,7 @@
 
 #include "third_party/widevine/cdm/widevine_cdm_common.h"
 
+#define WIDEVINE_CDM_VERSION_STRING "unknown"
 #define WIDEVINE_CDM_AVAILABLE
 
 #endif  // WIDEVINE_CDM_VERSION_H_
